src/components/operations/operation-details/ko/runtime/graphql-console.ts

```diff
@@ -529,4 +553,153 @@ export class GraphqlConsole {
+        (<any>window).monaco.editor.getModels().forEach(model => model.dispose());
+    }
+
+    public async closeWsConnection(): Promise<void> {
+            if(this.wsConnected()) {
+                this.wsConnected(false);
+                this.ws?.disconnect();
+            }
+    }
+
+    public async closeWs(): Promise<void> {
+        this.closeWsConnection();
+        this.displayWsConsole(false);
+    }
+
+    public async wsConnect(): Promise<void> {
+        this.editorValidations();
+        if (this.wsConnected() || this.editorErrors().length > 0) {
+            return;
+        }
+        
+        this.wsProcessing(true);
+        this.displayWsConsole(true);
+
+        let url = this.operationUrl();
+
+        if (url.startsWith("https://")) {
+            url = "wss://" + url.substring(8);
+        } else if (url.startsWith("http://")) {
+            url = "ws://" + url.substring(7)
+        }
+
+        this.initWebSocket();
+        this.ws.connect(url, graphqlSubProtocol);
+    }
+
+    public clearWsLogs(): void {
+        this.wsLogItems([]);
+    }
+
+    private initWebSocket(): void {
+        this.ws = new WebsocketClient();
+        this.ws.onOpen = () => {
+            this.wsProcessing(false);
+            this.wsConnected(true);
+            this.ws.send(JSON.stringify({
+                type: "connection_init",
+                payload: {}
+            }));
+            this.ws.send(JSON.stringify({
+                type: "subscribe",
+                id: "1",
+                payload: {
+                    query: this.document(),
+                    variables: this.variables() && this.variables().length > 0 ? JSON.parse(this.variables()) : null
+                }
+            }, null, 2));
+        };
+        this.ws.onLogItem = (data: LogItem) => {
+            if (!data) {
+                return;
+            }
+            if (data.logType === LogItemType.GetData) {
+                try {
+                    let json = JSON.parse(data.logData);
+                    if (json["type"] == "next" && "payload" in json) {
+                        data.logData = JSON.stringify(json["payload"], null, 2);
+                        this.wsLogItems.unshift(data);
+                    }
+                } catch (err) {
+                    console.log(err);
+                }
+            }
+            else if (data.logType === LogItemType.SendData) {
+                try {
+                    let json = JSON.parse(data.logData);
+                    if (json["type"] == "subscribe" && "payload" in json) {
+                        data.logData = JSON.stringify(json["payload"], null, 2);
+                        this.wsLogItems.unshift(data);
+                    }
+                } catch (err) {
+                    console.log(err);
+                }
+            }
+            else if (data.logType === LogItemType.Connection) {
+                if(_.includes(data.logData, 'Disconnected')) {
```

Comment:
`if(_.includes(data.logData, 'Disconnected'))` - when possible, please use native JS functions instead of lowdash: `data.logData.includes("Disconnected")`